TestbedScripts/TestEnvUtils.ps1

```diff
@@ -45,12 +45,29 @@ function Initialize-DockerNetwork {
 
 function Get-MacOfContainer {
     Param ([Parameter(Mandatory = $true)] [string] $ContainerName)
+    Write-Host -NoNewline "Reading MAC address of '$ContainerName'... "
     $Mac = docker exec $ContainerName powershell -Command {
         (Get-NetAdapter | Select-Object -First 1).MacAddress
     }
+    Write-Host $Mac
     return $Mac
 }
 
+function Get-FriendlyNameOfContainer {
+    Param ([Parameter(Mandatory = $true)] [string] $ContainerName)
+    Write-Host -NoNewline "Reading interface name of '$ContainerName'... "
+    $Name = docker exec $ContainerName powershell -Command {
+        (Get-NetAdapter | Select-Object -First 1).Name
+    }
+    if ($Name -match "\S+\s\((?<FriendlyName>[\S\s]+)\)") {
+        Write-Host $matches.FriendlyName
+        return $matches.FriendlyName
+    } else {
+        Write-Host "Failed."

```

Comment:
I think it would be beneficial to also print the `$Name` if regex match, so that it's easier to debug.